content/nginx-one-console/agent/configure-instances/configure-agent-features.md

```diff
@@ -87,3 +89,16 @@ Define features in the `nginx-agent.conf` file:
 
 Once the steps have been completed, users will be able to view metrics data being sent but will not have the capability to push NGINX configuration changes.
 
+{{<call-out "note" "Note: NGINX Access Log Metrics" >}} For collecting metrics from NGINX access logs, ensure that following log format is used in the NGINX configuration:
```

Comment:
I think this should be placed in this page instead https://frontdoor-test-docs.nginx.com/previews/docs/1520/nginx-one-console/getting-started/#enable-nginx-metrics-reporting